datastore/db/bytebase/FertiScan/schema_0.0.9.sql

```diff
@@ -0,0 +1,252 @@
+    "manufacturer_info_id" uuid REFERENCES "fertiscan_0.0.9".organization_information(id)
+    );
+
+    CREATE TYPE metric_type as ENUM ('volume', 'weight','density');
+
+    CREATE TABLE "fertiscan_0.0.9"."metric" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "value" float NOT NULL,
+    "edited" boolean,
+    "unit_id" uuid REFERENCES "fertiscan_0.0.9".unit(id),
+    "metric_type" metric_type,
+    "label_id" uuid REFERENCES "fertiscan_0.0.9".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."sub_type" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "type_fr" text Unique NOT NULL,
+        "type_en" text unique NOT NULL
+    );
+
+    -- CREATE A TYPE FOR FRENCH/ENGLISH LANGUAGE
+    CREATE TYPE LANGUAGE AS ENUM ('fr', 'en');
+
+    CREATE TABLE "fertiscan_0.0.9"."specification" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "humidity" float,
+    "ph" float,
+    "solubility" float,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.9".label_information(id),
+    "language" LANGUAGE
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."sub_label" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "text_content_fr" text NOT NULL DEFAULT '',
+        "text_content_en" text NOT NULL DEFAULT '',
+        "label_id" uuid NOT NULL REFERENCES "fertiscan_0.0.9"."label_information" ("id"),
+        "edited" boolean NOT NULL,
+        "sub_type_id" uuid NOT NULL REFERENCES "fertiscan_0.0.9"."sub_type" ("id")
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."micronutrient" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "read_name" text NOT NULL,
+    "value" float NOT NULL,
+    "unit" text NOT NULL,
+    "element_id" int REFERENCES "fertiscan_0.0.9".element_compound(id),
+    "label_id" uuid REFERENCES "fertiscan_0.0.9".label_information(id),
+    "edited" boolean,
+    "language" LANGUAGE
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."guaranteed" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "read_name" text NOT NULL,
+    "value" float NOT NULL,
+    "unit" text NOT NULL,
+    "element_id" int REFERENCES "fertiscan_0.0.9".element_compound(id),
+    "label_id" uuid REFERENCES "fertiscan_0.0.9".label_information(id),
+    "edited" boolean
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."ingredient" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "organic" boolean,
+    "active" boolean,
+    "name" text NOT NULL,
+    "value" float,
+    "unit" text,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.9".label_information(id),
+    "language" LANGUAGE
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."inspection" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "verified" boolean DEFAULT false,
+    "upload_date" timestamp DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" timestamp DEFAULT CURRENT_TIMESTAMP,
+    "inspector_id" uuid REFERENCES "fertiscan_0.0.9".users(id),
```

Comment:
Loosely enforcing the constraints is fine for testing. But eventually the `one and only one` relations from the erd need to be enforced with "NOT NULL" in the schema code.

Reply:
It should be not null indeed

Reply:
There are such other relations that we haven't strictly enforced. Could be done in another issue if you want. Resolving this for now.